src/geometry.rs

```diff
@@ -754,6 +807,2036 @@ mod tests {
         (faces, vertices, normals)
     }
 
+    fn torus() -> (Vec<(u32, u32, u32)>, Vec<Point3<f32>>) {
```

Comment:
Please move all these data fixtures to `test_geometry_fixtures.rs` and add this text on top:

```
// FIXME: move to data fixture files once we are able to serialize and deserialize stuff.
```